clang/test/CXX/drs/cwg28xx.cpp

```diff
@@ -153,6 +153,8 @@ void g() {
 namespace cwg2881 { // cwg2881: 19 ready 2024-06-26
 
 #if __cplusplus >= 202302L
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Winaccessible-base"
```

Comment:
This seems rather strange to me. Why are we turning off diagnostics in a test?

Reply:
Yes, we shouldn't be using `#pragma clang diagnostic` in DR tests. I'll get rid of the few that are left.
If a diagnostic (ideally) should not be emitted, match it and leave a FIXME. If diagnostic is correct, you can try to change the test in a way that doesn't trigger it, but it's not required.